.github/workflows/_wfc_dotnet-ci-build.yml

```diff
@@ -17,210 +27,45 @@ on:
-        working-directory: src
-      - name: Report Integration Test Coverage
-        run: |
-          $opencoverxmlpath = Get-ChildItem 'artifacts/inttestcoverage/coverage.xml' | Select-Object -First 1
-          $intTestCoverage = (Select-Xml -Path $opencoverxmlpath -XPath 'CoverageSession/Summary/@branchCoverage').Node.Value
-          $coverageMessage = "Integration Test Coverage: $intTestCoverage"
-          echo "::notice::$coverageMessage"
-          Add-Content -Path $env:GITHUB_STEP_SUMMARY -Value $coverageMessage
-      - name: Produce Nuget Packages
-        run: |
-          dotnet pack ${{ env.solutionpath }} --configuration Release --no-build /bl:${{ github.workspace }}\artifacts\binlog\pack.binlog --nologo /p:PackageOutputPath=${{ github.workspace }}\artifacts\nuget /p:ContinuousIntegrationBuild=true
-        working-directory: src
-        env:
-          CHANGE_COMMIT_LOG: ${{ steps.changelog.outputs.commitLog }}
-      - name: Generate SBOM
-        run: |
-          dotnet sbom-tool generate -b artifacts/nuget -bc src -pn ${{ inputs.solutionName }} -pv $Env:NBGV_SimpleVersion -ps "DHGMS Solutions" -nsb https://sbom.dhgms.com -m artifacts\sbom
-      - name: List outdated packages
-        run: |
-          dotnet outdated -o ${{ github.workspace }}\artifacts\outdated\outdated.json
-        working-directory: src
-      - name: VirusTotal Monitor Scan
-        if: ${{ env.VIRUSTOTAL_API_KEY != 0 }}
-        uses: crazy-max/ghaction-virustotal@v4
-        with:
-          vt_api_key: ${{ env.VIRUSTOTAL_API_KEY }}
-          vt_monitor: true
-          monitor_path: /ghaction-virustotal
-          files: |
-            ./artifacts/nuget/*.nupkg
-        env:
-          VIRUSTOTAL_API_KEY: ${{ secrets.VIRUSTOTAL_API_KEY }}
-      - name: store binlogs
-        if: ${{ failure() }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: binlogs
-          path: artifacts/binlog
-      - name: store unit test coverage
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: unittestcoverage
-          path: artifacts/unittestcoverage
-          if-no-files-found: error
-      - name: store nuget packages
-        uses: actions/upload-artifact@v4
-        with:
-          name: nuget
-          path: artifacts/nuget
-          if-no-files-found: error
-      - name: store omd
-        uses: actions/upload-artifact@v4
-        with:
-          name: omd
-          path: artifacts/omd
-      - name: store sbom
-        uses: actions/upload-artifact@v4
-        with:
-          name: sbom
-          path: artifacts/sbom
-      - name: store outdated
-        uses: actions/upload-artifact@v4
-        with:
-          name: outdated
-          path: artifacts/outdated
+  build-linux:
+    if: inputs.buildOs == 'linux'
```

Comment:
always going to build the linux version. it's just whether it's the primary os for pack and publish etc.